docs/content.go

```diff
@@ -715,10 +715,39 @@ Enterprise Performance Computing (EPC)`
 	InspectUse   string = `inspect [inspect options...] <image path>`
 	InspectShort string = `Show metadata for an image`
 	InspectLong  string = `
-  Inspect will show you labels, environment variables, and scripts associated 
-  with the image determined by the flags you pass.`
+  Inspect will show you labels, environment variables, apps and scripts associated 
+  with the image determined by the flags you pass.
+  `
 	InspectExample string = `
-  $ singularity inspect ubuntu.sif`
+  $ singularity inspect ubuntu.sif
+  
+  If you want to list the applications (apps) installed in a container (located at
+  /scif/apps) you should run inspect command with --list-apps <container-image> flag.
+  ( See https://sci-f.github.io for more information on SCIF apps)
+
+  The following environment variables are available to you when called 
+  from the shell inside the container. The top variables are relevant 
+  to the active app (--app <app>) and the bottom available for all 
+  apps regardless of the active app:
+  ACTIVE APP ENVIRONMENT:
+      SCIF_APPNAME       the name of the application
+      SCIF_APPROOT       the application base (/scif/apps/<app>)
+      SCIF_APPMETA       the application metadata folder
+      SCIF_APPDATA       the data base folder for active app
+        SCIF_APPINPUT    expected input folder within data base folder
+        SCIF_APPOUTPUT   the output data folder within data base folder
+  GLOBAL APP ENVIRONMENT:
```

Comment:
A few are missing here (and double check they are defined for Singularity).

 - `SCIF_APPENV` points to the application's environment.sh file, so for example, you could source it without knowing the entire path.
 - `SCIF_APPLABELS`: is the labels.json in the metadata folder, if the user wanted to parse it.
 - `SCIF_APPBIN: is the bin folder for the app, which is automatically added to the $PATH when the app is active.
 - `SCIF_APPLIB` is the lib folder, same deal, but added to `LD_LIBRARY_PATH` 
 - `SCIF_APPRUN`: is the runscript
 - `SCIF_APPHELP`: is the help file for the runscript (what you get when you do help <appname>
 - `SCIF_APPTEST`: is the test.sh (testing) script that you run when the user does test <appname>